packages/tidy-moment/src/summarizeMomentGranularity.ts

```diff
@@ -56,6 +56,12 @@ export function summarizeMomentGranularity<
     } = options;
 
     const dateFormats = {
+      s: 'YYYY-MM-DDThh-mm-ss',
+      seconds: 'YYYY-MM-DDThh-mm-ss',
+      second: 'YYYY-MM-DDThh-mm-ss',
+      min: 'YYYY-MM-DDThh-mm',
+      minutes: 'YYYY-MMTDD-hh-mm',
+      minute: 'YYYY-MMTDD-hh-mm',
```

Comment:
Can we just have these be typical formats, even though it doesn't really matter. I find the dashes just a bit surprising. e.g. `YYYY-MM-DDTHH:mm:ss`. Also prefer 24-hour `HH` over `hh`. And make the min/minutes/minute ones the same (the T got moved)

Reply:
Done 👌